lib/linter.coffee

```diff
@@ -2,6 +2,7 @@ fs = require 'fs'
 path = require 'path'
 {CompositeDisposable, Range, Point, BufferedProcess} = require 'atom'
 XRegExp = null
+Grim = require('grim')
```

Comment:
I meant that you can use destructuring, we don't need that Grim variable for any other purposes than deprecate. Then how about you extract that variable from it on require time